stainless_extraction/src/ty.rs

```diff
@@ -48,6 +48,19 @@ pub(super) struct Generics<'l> {
   pub trait_bounds: Vec<&'l st::ClassType<'l>>,
 }
 
+#[inline]
+fn usize_bit_width(tcx: TyCtxt<'_>) -> u64 {
```

Comment:
Might be worth adding a comment to point to https://rust-lang.github.io/unsafe-code-guidelines/layout/scalars.html?highlight=usize#isize-and-usize

Reply:
Let's also rename it to something like `pointer_bit_width`?